.gitignore

```diff
@@ -26,3 +26,9 @@ cmake-build-*
 # mavlink repo to generate mavlink-headers
 mavlink
 mavlink-headers
+
+
+# vitepress - Docs guide
+node_modules/
```

Comment:
Would be nice if we can move the `node_modules` into docs.

Reply:
Done. We just have to build from docs rather than root.